File: specs/Qowaiv.CodeAnalysis.Specs/ParseShouldNotFail_specs.cs
namespace Rules.ParseShouldNotFail_specs;

public class Verify
{
    [Test]
    public void Rule()
        => new ParseShouldNotFail()
        .ForCS()
        .AddSource(@"Cases/ParseShouldNotFail.cs")
        .AddReference<Qowaiv.Percentage>()
        .Verify();
}

File: specs/Qowaiv.CodeAnalysis.VisualBasic.Specs/ParseShouldNotFail_specs.vb
Imports NUnit.Framework
Imports CodeAnalysis.TestTools
Imports Qowaiv.CodeAnalysis

Namespace ParseShouldNotFail_specs

    Public Class Verify

        <Test>
        Public Sub VisualBasic()
            Dim rule As New ParseShouldNotFail()
            rule.ForVB().
                AddSource("Cases/ParseShouldNotFail.vb").
                AddReference(Of Qowaiv.Percentage).
                Verify()
        End Sub

    End Class

End Namespace
